python/instrumentation/openinference-instrumentation-crewai/src/openinference/instrumentation/crewai/_wrappers.py

```diff
@@ -205,15 +212,36 @@ def __call__(
                 ),
             )
             try:
-                response = wrapped(*args, **kwargs)
+                crew_output = wrapped(*args, **kwargs)
+                usage_metrics = instance.usage_metrics
```

Comment:
Seems like `crew=instance`, should we use one or the other?

Reply:
good catch!